tests/drivers/gpio/gpio_nrf/src/main.c

```diff
@@ -58,4 +60,37 @@ ZTEST(gpio_nrf, test_gpio_high_drive_strength)
 		err);
 }
 
+/*
+ * Nordic Semiconductor specific,
+ * gpio manipulation with disabled NRFX interrupts
+ */
+ZTEST(gpio_nrf, test_gpio_manipulation_nrfx_int_disabled)
```

Comment:
Where is `GPIO_NRFX_INTERRUPT` disabled for this testcase?

Reply:
Updates